### ISSUE TITLE
Improve UX for organize imports on save failures / cancellations 

### ISSUE THREAD

**Issue by dmytro-gokun** [microsoft/vscode#75792]
Version: 1.35.1 (user setup)
Commit: c7d83e57cd18f18026a8162d042843bda1bcf21f
Date: 2019-06-12T14:30:02.622Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362

I'm trying to execute "Organize imports" and getting this in the console (the imports are not affected):

[2019-06-19 12:48:30.689] [renderer1] [warning] Aborted codeActionsOnSave after 750ms

Question is.. What is that? Should i increase this timeout somehow? Is there a way to find out why it times out?

I also think it should not just fail silently like it does today (only producing a record in the console), leaving me wondering why my code is not getting formatted..

**Comment by mjbvz** [comment]
Does this still happen if you disable all extensions?

Also, does this happen for every file/project, or only in specific files or projects?

**Comment by dmytro-gokun** [comment]
@mjbvz I've tried to find some patterns, but failed to do so. Mostly it works, when it fails, it does so silently. So it's pretty hard to notice.  I'm not sure what to do about, but i think it would really help to have the following:

1. A setting to "escalate" the level or the error. It should produces some kind of a modal popup which is impossible to miss.
2. When it fails, it should be possible to obtain some kind of "dump". What exactly it has been processing, which extensions were loaded etc.

With this tools, it will be possible to notice & debug the issue. In the hindsight, i realized that i saw this behavior a lot in the past. I have this configured onSave and for several month now i've noticed some files not having imports organized. But since it was not producing a big fat popup, i was unaware of it. I had to realize it was not working, then i went searching for the correct console where i saw the message. What i'm think may happen is that this thing actually fails a lot for different people (i saw it two other person's computers, as i realize now). But it fails silently and that's why nobody complains.
